src/odin/narrativebuilder.cc

```diff
@@ -4925,18 +4924,5 @@ std::string NarrativeBuilder_ruRU::GetPluralCategory(size_t count) {
   }
   return kPluralCategoryOtherKey;
 }
-
-std::string NarrativeBuilder::FormBssManeuverType(DirectionsLeg_Maneuver_BssManeuverType type) {
```

Comment:
It's indeed a makeshift solution at the time. I didn't have time to finish it properly. I think the proper way is to put it in the "translations" part.

Reply:
i agree with @xlqian we can simply add this to our translations and pull it out of that file to use here. When we do that we can also just immediately do the french translation so that it will already be complete 😄

Reply:
i forget what other combo phrases we have but we might also want to follow that model rather than the `phrase1 + phrase2` thing we have above too but maybe that is the way we do it in other scenarios, we'll see!

Reply:
Ok, I’ll take a look and make you a proposal.